_data/investing.yml

```diff
@@ -57,6 +57,12 @@ websites:
       software: Yes
       doc: https://www.interactivebrokers.com/en/index.php?f=ibgStrength&p=log
 
+    - name: John Hancock
+      url: https://www.jhpensions.com
```

Comment:
@Jawshy yep! jhpensions.com 301s me to that full url, apologies for the issues. Will fix.